dist.ini

```diff
@@ -14,8 +14,10 @@ copyright_holder = Shawn M Moore
 [AutoPrereqs]
 
 [Prereqs]
-Any::Moose = 0.12
+Moo = 0.12
 Try::Tiny  = 0.02
+Type::Tiny::Class = 1
```

Comment:
These additions are not necessary, due to `[AutoPrereqs]`. Also, "1" isn't any specific release version of this distribution, is it? If a specific version is required, it should be explicitly stated.

Reply:
Sorted